test/tc_basic.rb

```diff
@@ -114,7 +114,7 @@ def test_set_and_get_point_from_wkt
               obj_ = klass_.new
               assert_nil(obj_.latlon)
               obj_.latlon = 'SRID=1000;POINT(1 2)'
-              assert_equal(@factory.point(1, 2), obj_.latlon)
+              assert_equal(RGeo::Cartesian.preferred_factory(srid: 1000).point(1, 2), obj_.latlon)
```

Comment:
@januszm I'm at a loss as to where to look. I'm on macOS 10.12.5 with current XCode and mysql 5.6.30. I'm using Ruby 2.3.4 but saw the same results on a couple of other versions. I delete and recreate gemsets and `bundle clean --force` often, so there are no gems or gem versions in my gemset other than those in that Gemfile.lock.

Just to be clear, I meant that I see the failures on my branch (whose gemspec results in that Gemfile.lock) when I roll back my change to tc_basic.rb. But you probably already understood that, because why else would I have made the changes?